### DISCUSSION TITLE
3D error with AMD GPU

### DISCUSSION THREAD

**Discussion by qwadroxxx** [deforum-art/deforum-stable-diffusion#266] [General]
Generation of 2D images works correctly, but when I try to do 3D generation, only the first image is done successfully, and the rest of the images are gray, I tried a lot of things, nothing helps.
AMD RX 6900 XT

**Comment by ai-d-g** [comment]
Apparently AMD doesn't work in 3D because the script is written specifically for Nvidia Cuda.
